doc/amazon_sagemaker_operators_for_kubernetes.rst

```diff
@@ -82,12 +82,12 @@ IAM role-based operator deployment
 Before you can deploy your operator using an IAM role, associate an OpenID Connect (OIDC) provider with your role to
 authenticate with the IAM service.
 
-Associate an OpenID Connect Provider to Your Instance
+Create an OpenID Connect Provider to Your Instance
```

Comment:
"Create `x` to `y`" doesn't make sense. Change to "Create `x` for `y`", e.g. `Create an OpenID Connect Provider for Your Cluster`?